### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by jugosack**
In this pull request, we completed the following project requirements:

- Delete all the boilerplate from Create React App (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches the design
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use functional components
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+.container {
+  margin: auto;
+  width: 30%;
+  height: 70%;
+  padding: 10px;
+  border: 1px solid #f2f2f2;
+  -webkit-box-shadow: 1px 1px 4px 0 rgba(176, 176, 176, 1);
+  -moz-box-shadow: 1px 1px 4px 0 rgba(176, 176, 176, 1);
+  box-shadow: 1px 1px 4px 0 rgba(176, 176, 176, 1);
```
Comment (omar-muhamad) [inline src/components/Calculator.scss:13]: - [ ] Great job using SASS module for styling.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import './Calculator.scss';
+
+function Calculator() {
+  return (
+    <>
+      <div className="space" />
```
Comment (omar-muhamad) [inline src/components/Calculator.js:6]: - [ ] good job using react fragment here.

File: src/components/Calculator.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  background-color: #f2f2f2;
+}
+
+#plus,
+#slash,
+#ex,
+#minus,
+#equal {
+  background-color: bisque;
+}
```
Comment (omar-muhamad) [inline src/components/Calculator.scss]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A), Please change the background color of the operators buttons here to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image |

Hint: You can use this color `#F5913E`

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @jugosack,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Calculator.scss (hunk 3 above)
Comment (jugosack) [reply src/components/Calculator.scss]: @Omar-Muhamad Thanks for your suggestion and especially hint 💯

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @jugosack,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**